pycolmap/pyproject.toml

```diff
@@ -2,31 +2,29 @@
 requires = ["scikit-build-core>=0.3.3", "pybind11==2.12.0"]
 build-backend = "scikit_build_core.build"
 
-
 [project]
 name = "pycolmap"
 version = "3.11-dev"
-description="COLMAP bindings"
+description = "COLMAP bindings"
 readme = "README.md"
 authors = [
+  { name = "Johannes Schönberger", email = "jsch@demuc.de" },
```

Comment:
@B1ueber2y Would you like to be added here? What details should I use?

Reply:
Thanks. I would like to. Could we use ``b1ueber2y@gmail.com``. This is my personal mail address that I have been using for a long time.